ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.11-presubmits.yaml

```diff
@@ -0,0 +1,235 @@
+presubmits:
+  openshift/windows-machine-config-bootstrapper:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^community-4\.11$
+    - ^community-4\.11-
+    cluster: build01
+    context: ci/prow/okd-build
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/variant: okd
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-windows-machine-config-bootstrapper-community-4.11-okd-build
+    rerun_command: /test okd-build
+    skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
nit since this is WMCB config:

Consider using the updated regex introduced here https://github.com/openshift/release/pull/31465/commits/66a6b2d0155df4107d4a61228e8bd615ead12592 and update the commit message. Applies throughout the commit